ext/ffi/tests/gh10568.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+GH-10568 (Assertion failure when var_dump'ing void FFI result)
+--EXTENSIONS--
+ffi
+--SKIPIF--
+<?php
+try {
+    $libc = FFI::cdef("void strlen(const char *s);", "libc.so.6");
+} catch (Throwable $_) {
+    die('skip libc.so.6 not available');
+}
```

Comment:
```suggestion
} catch (Throwable) {
    die('skip libc.so.6 not available');
}
```
You can omit the variable as of PHP 8.0

Reply:
Thanks, I'll wait for other reviews to come in and then I'll update this